packages/stockflux-watchlist/src/components/watchlist/Watchlist.js

```diff
@@ -86,6 +89,11 @@ const Watchlist = () => {
 
   const getSymbolIndex = symbol => watchlist.indexOf(symbol);
 
+  const onDropOutside = (symbol, stockName) => {
+    // the 3rd param is for intents enabled.
+    Launchers.launchChart(symbol, stockName, null, previewPosition);
```

Comment:
Again, just a suggestion, but since it's very unlikely that we will be using intents soon, I would suggest reordering arguments so that the intents flag would be the very last argument.

Reply:
Reordered the inputs to `Launchers.launchChart` to have position before intents. Added parameters to the button shortcut for chart to allow you to pass in positions, this prevents having a `null` argument in `app-shortcutes/Chart.js`